app/services/cloud_front_header_parser.rb

```diff
@@ -10,6 +10,7 @@ def client_port
 
   # Source IP and port for client connection to CloudFront
   def viewer_address
+    return nil unless @request&.headers # guard against request being nil
```

Comment:
```suggestion
    return nil unless @request&.headers
```

This was a pointless comment.